backend/functions/lib/controllers/case-history/case-history.controller.ts

```diff
@@ -0,0 +1,50 @@
+import { ApplicationContext } from '../../adapters/types/basic';
+import { CaseHistoryUseCase } from '../../use-cases/case-history/case-history';
+import { CamsError } from '../../common-errors/cams-error';
+import { UnknownError } from '../../common-errors/unknown-error';
+import { CaseAssignmentHistory } from '../../adapters/types/case.assignment';
+
+const MODULE_NAME = 'CASE-HISTORY-CONTROLLER';
+
+interface SuccessMonad {
+  success: true;
+  body: unknown;
+}
+
+interface ErrorMonad {
+  success: false;
+  message: string;
+  errors: Array<string>;
+}
+
+type Monad = SuccessMonad | ErrorMonad;
```

Comment:
Monad, SuccessMonad and ErrorMonad are now repeatedly defined in the codebase. We should move them to a common lib. We should also consider how they are used and how we want to define standard response shapes from the API.